kibana.tf

```diff
@@ -0,0 +1,31 @@
+module "kibana_label" {
+  source  = "justtrackio/label/null"
+  version = "0.26.0"
+
+  label_order      = var.label_orders.kibana
+  label_value_case = "title"
+
+  enabled   = var.kibana_data_view_enabled
+  delimiter = " "
+
+  context = module.this.context
+}
+
+resource "elasticstack_kibana_space" "default" {
+  count = var.kibana_space_enabled ? 1 : 0
+
+  name     = "${module.this.environment} ${module.this.stage}"
+  space_id = "${module.this.environment}-${module.this.stage}"
+}
```

Comment:
The issue here is that we only can create the space with one app and the others would need to skip it.

Reply:
omitting the space for now, as this should happen on a higher level